nbextensions/usability/toc2/main.js

```diff
@@ -146,26 +147,14 @@ define(["require", "jquery", "base/js/namespace",  'services/config',
     $([IPython.events]).on("notebook_loaded.Notebook", function(){ // curiously, the event is not always fired or detected
```

Comment:
FYI, the event _is_ always fired, the issue is that depending on the notebook's complexity, the connection speed & lag time, it's possible for the notebook to load _before_ the extension is loaded, in which case you miss the event.